DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/lacp/test_lacp_blocked_port_stp_rstp.py

```diff
@@ -0,0 +1,164 @@
+import asyncio
+import pytest
+
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.lib.mstpctl.mstpctl import Mstpctl
+from dent_os_testbed.utils.test_utils.tgen_utils import tgen_utils_get_dent_devices_with_tgen
+
+pytestmark = [pytest.mark.suite_functional_devlink,
```

Comment:
```suggestion
pytestmark = [pytest.mark.suite_functional_lacp,
```
Please make sure that you change this in all of your tests
```

<Package lacp>
  <Module test_lacp_blocked_port_stp_rstp.py>
    <Function test_lacp_root_port[rstp]>
    <Function test_lacp_root_port[stp]>
  <Module test_lacp_loopback_detection.py>
    <Function test_lacp_loopback_detection[rstp]>
    <Function test_lacp_loopback_detection[stp]>
  <Module test_lacp_root_bridge_stp_rstp.py>
    <Function test_lacp_root_bridge_selection_stp[stp]>
    <Function test_lacp_root_bridge_selection_stp[rstp]>
  <Module test_lacp_traffic_during_topology_convergence.py>
    <Function test_lacp_traffic_during_topology_convergence[stp]>
    <Function test_lacp_traffic_during_topology_convergence[rstp]>
```